Modules/_functoolsmodule.c

```diff
@@ -335,8 +335,9 @@ partial_call(partialobject *pto, PyObject *args, PyObject *kwargs)
 }
 
 PyDoc_STRVAR(partial_doc,
-"partial(func, *args, **keywords) - new function with partial application\n\
-    of the given arguments and keywords.\n");
+"partial(func, /, *args, **keywords)\n--\n\n\
+Create a new function with partial application\n\
+of the given arguments and keywords.\n");
```

Comment:
```suggestion
Create a new function with partial application of the given arguments and keywords.\n");
```

Reply:
It will be too long line. It should be wrapped. Or re-worded.